packages/itwin/property-grid/api/property-grid-react.api.md

```diff
@@ -236,7 +237,7 @@ export type PropertyGridWidgetProps = PropertyGridComponentProps & PropertyGridW
 export function RemoveFavoritePropertyContextMenuItem({ field, imodel, scope, onSelect }: FavoritePropertiesContextMenuItemProps): JSX.Element | null;
 
 // @public (undocumented)
-type SelectionStorage = Pick<SelectionStorage_2, "getSelection" | "replaceSelection" | "selectionChangeEvent">;
+type SelectionStorage_2 = Pick<SelectionStorage, "getSelection" | "replaceSelection" | "selectionChangeEvent">;
```

Comment:
We used to take `Pick<SelectionStorage, "getSelection" | "replaceSelection" | "selectionChangeEvent">`, now we're taking the full `SelectionStorage`. Probably not intentional?

Reply:
We are passing it to `usePropertyDataProviderWithUnifiedSelection` and it takes full `SelectionStorage`.
https://github.com/iTwin/presentation/blob/e56e47016a165aafe044f95682ed4abd1b54ac0d/packages/components/src/presentation-components/propertygrid/UseUnifiedSelection.tsx#L45